os/kernel/init/os_start.c

```diff
@@ -330,21 +330,25 @@ const struct tasklist_s g_tasklisttable[NUM_TASK_STATES] =
 	{                                              /* TSTATE_WAIT_MQNOTFULL */
 		&g_waitingformqnotfull,
 		TLIST_ATTR_PRIORITIZED
+	},
+#else
+	{                                             
+		NULL,
+		0
```

Comment:
`#ifndef` has two lists, but `#else` has just one list. One more list should added with null value.